Providers/IntercomProvider.m

```diff
@@ -33,7 +33,7 @@ - (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email
     }
 }
 
-- (void)setUserProperty:(NSString *)property toValue:(NSString *)value
+- (void)setUserProperty:(NSString *)property toValue:(id)value {
```

Comment:
Hah, and it’s the _one_ that I could have catched by running the tests locally :smile: